lib/luaT/luaT.c

```diff
@@ -471,16 +471,15 @@ void luaT_registeratname(lua_State *L, const struct luaL_Reg *methods, const cha
 }
 
 
-/* utility functions */
-const char *luaT_classrootname(const char *tname)
+const char* luaT_classrootname(const char *tname)
```

Comment:
Can you update the documentation for this method? Also, what about `luaT_classmodulename`? If you pass in `a.b.c` do you want it to return `a` (as it currently does) or `a.b`?